#! /bin/sh /usr/share/dpatch/dpatch-run
## 220-magic-update-erlang.dpatch by Colin Watson <cjwatson@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Update Erlang magic to avoid misidentifications (LP: #248619).
## DP: Backported from upstream/Debian.

@DPATCH@
diff -urNad file-4.26~/magic/Magdir/erlang file-4.26/magic/Magdir/erlang
--- file-4.26~/magic/Magdir/erlang	2008-02-19 17:27:29.000000000 +0000
+++ file-4.26/magic/Magdir/erlang	2009-06-04 12:25:13.000000000 +0100
@@ -12,7 +12,7 @@
 >8	string	BEAM		Erlang BEAM file
 
 # 4.2 version may have a copyright notice!
-4	string	Tue Jan 22 14:32:44 MET 1991	Erlang JAM file - version 4.2
-79	string	Tue Jan 22 14:32:44 MET 1991	Erlang JAM file - version 4.2
+4	string	Tue\ Jan\ 22\ 14:32:44\ MET\ 1991	Erlang JAM file - version 4.2
+79	string	Tue\ Jan\ 22\ 14:32:44\ MET\ 1991	Erlang JAM file - version 4.2
 
-4	string	1.0 Fri Feb 3 09:55:56 MET 1995	Erlang JAM file - version 4.3
+4	string	1.0\ Fri\ Feb\ 3\ 09:55:56\ MET\ 1995	Erlang JAM file - version 4.3
